### ISSUE TITLE
PS3 controller mapping

### ISSUE THREAD

**Issue by JamesTheAwesomeDude** [libretro/retroarch-joypad-autoconfig#54]
The PS3 controller has X, the confirm/jump/accept/yes button (equivalent in every aspect to the conventional "A" button) on the bottom of the ABXY pad. This has caused it to be incorrectly mapped to B on the RetroPad.

Likewise, Circle, which is red, and functions as the no/back/cancel button, has been mapped to the RetroPad's A button due to being on the right side of the ABXY pad.

I don't play enough PS3 games to confirm this, but I highly suspect that Sony has the Square and Triangle buttons, too, in mirrored positions from their XBox controller counterparts (X and Y).

The tl;dr is: Shouldn't the PS3 controller be mapped by default based on what the buttons do instead of on their position? Having, for example, the universally acknowledged Cancel/Back button mapped to A doesn't make a lot of sense.

**Comment by JamesTheAwesomeDude** [comment]
I can make the changes myself - would a PR be welcome?

**Comment by twinaphex** [comment]
I'm sorry but this is unwanted.

The PlayStation pad is just a SNES pad given a minor facelift. The face buttons should behave exactly as the SNES pad did in terms of layout. Right now they are just fine.
